src/Features/header/components/CityLanguageSelect/City/CitySelect.tsx

```diff
@@ -0,0 +1,81 @@
+import Link from 'next/link';
+import React, { ReactElement, useState, useEffect } from 'react';
+import { setCookie, getCookie, CookieValueTypes } from 'cookies-next';
+import { Modal } from '../../../../../components/Modal/Modal';
+
+interface CitySelectProps {
+  myValue: string | undefined;
+}
+
+export const CitySelect: React.FC<CitySelectProps> = ({ myValue }): ReactElement => {
+  const [showModal, setShowModal] = useState<boolean>(false);
```

Comment:
Don't use `<boolean>` or other generics for primitive types like string, number, etc. Use it only for custom objects